### PR TITLE
Review my code

### PR DESCRIPTION
**Pull request by GeorgesG9**
desc

### CODE REVIEW

File: TestProject/Program.cs [hunk 1]
```diff
@@ -8,6 +8,8 @@ static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
             Console.WriteLine("Commit new branch dev - Joseph ");
+            Console.WriteLine("Commit new branch devGeo - Georges ");
```
Comment (josephakl) [inline TestProject/Program.cs:11]: Okay for new branch devGeo

File: TestProject/Program.cs [hunk 2]
```diff
@@ -8,6 +8,8 @@ static void Main(string[] args)
...
+            Console.WriteLine("my new change - Georges ");
```
Comment (josephakl) [inline TestProject/Program.cs:12]: Okay new change

File: TestProject/Program.cs (hunk 1 above)
Comment (GeorgesG9) [reply TestProject/Program.cs:11]: thank you

**Review by josephakl — COMMENTED** [review]
Okay for new changes (branch dev and branch devGeo writlines)

**Review by josephakl — APPROVED** [review]
Approved changes

**Pull request merged**